src/lib/core/CHIPError.h

```diff
@@ -2410,6 +2410,14 @@ using CHIP_ERROR = ::chip::ChipError;
  */
 #define CHIP_ERROR_IM_MALFORMED_STATUS_RESPONSE_MESSAGE      CHIP_CORE_ERROR(0xe1)
 
+/**
+ * @def CHIP_ERROR_MESSAGE_COUNTER_EXHAUSTED
```

Comment:
Please add this in one of the existing slots where we have a free value, not at the end.